devicetypes/smartthings/zwave-lock.src/zwave-lock.groovy

```diff
@@ -631,6 +634,8 @@ private def handleBatteryAlarmReport(cmd) {
 	def map = null
 	switch(cmd.zwaveAlarmEvent) {
 		case 0x01: //power has been applied, check if the battery level updated
+			state.queryBattery = true
```

Comment:
See my comment about setting `queryBattery` to true here. and generating double Battery gets.